core/services/cron/validate.go

```diff
@@ -36,6 +38,10 @@ func ValidateCronSpec(tomlString string) (job.Job, error) {
 		return jb, errors.Errorf("the only supported schema version is currently 1, got %v", jb.SchemaVersion)
 	}
 
+	if !(strings.HasPrefix(spec.CronSchedule, "CRON_TZ=") || strings.HasPrefix(spec.CronSchedule, "@")) {
+		return jb, errors.New("cron schedule must specify a time zone using CRON_TZ, e.g. 'CRON_TZ=UTC 5 * * * *'")
```

Comment:
Maybe document the `@every` format in this error message?

Reply:
Done!